delegates/src/main/java/com/revolut/recyclerkit/delegates/DefaultDiffAdapter.kt

```diff
@@ -0,0 +1,130 @@
+package com.revolut.recyclerkit.delegates
+
+import android.os.Looper
+import androidx.annotation.UiThread
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import java.lang.ref.WeakReference
+
+/*
+ * Copyright (C) 2023 Revolut
+ * Copyright (C) 2014 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+/*
+* @param autoScrollToTop if autoscroll is true then RecyclerView will be scrolled to zero item on update
+* if last zero item was completely visible (i.e. zero scroll).
+* @param detectMoves – true if DiffUtil should try to detect moved items, false otherwise.
+*/
+
+open class DefaultDiffAdapter(
```

Comment:
`DefaultDiffAdapter` introduces a conflict with `DiffAdapter`. The naming doesn't suggest what's the difference between them.
We can merge their functionality together or come up with the more straightforward naming

Reply:
removed